collection-scripts/gather_network_logs

```diff
@@ -15,6 +15,20 @@ function gather_multus_data {
   oc get multi-networkpolicy -o yaml --all-namespaces > "${NETWORK_LOG_PATH}"/multi_networkpolicy 2>&1 & PIDS+=($!)
 }
 
+MULTUS_DAEMON_PODS="${@:-$(oc -n openshift-multus get pods -l app=multus -o jsonpath='{.items[*].metadata.name}')}"
```

Comment:
does this run by default?  if so, this must be restricted to masters.

Reply:
For the purposes of debugging the sandbox creation failures -- wouldn't we want the logs from all nodes?